src/main/ml-modules/services/geoQueryService.sjs

```diff
@@ -679,21 +679,7 @@ function getObjects(req, exportPlan=false) {
 
   // TODO: look into how to deal with object ids more generally
```

Comment:
Let's smoke this TODO - I don't know what it means, Sonar's not going to like it, I just don't think it's adding any value.